app/assets/stylesheets/alchemy/image_library.scss

```diff
@@ -66,17 +120,39 @@ $image-overlay-form-width: 350px;
   }
 }
 
-#zoomed_picture_background {
+.picture-overlay-handle {
+  width: $picture-overlay-handle-width;
+  height: 100%;
+  right: $image-overlay-form-width;
+  cursor: pointer;
+
+  .icon-angle-double-right {
+    position: absolute;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets<br>Properties should be ordered color, font-size, left, position, top